src/dev/create-extension.ts

```diff
@@ -28,13 +28,19 @@ const ncpPromise = promisify(ncp);
 const currentFileUrl = import.meta.url;
 const templateDirectory = path.resolve(decodeURI(fileURLToPath(currentFileUrl)), "../../../templates");
 
-const getProjectPath = (rawArgs: string[]) => {
-  const args = arg({}, { argv: rawArgs.slice(2) });
+const getParsedArgs = (rawArgs: string[]) => {
+  const args = arg(
+    {
+      "--from-scaffold-eth": String,
```

Comment:
silly thing: would something like `--from-hash` or `--from-commit` sound more natural?

Or do you want to be super explicit about it with "scaffold-eth"?

Reply:
Maybe use something like `--from-se2-commit`